src/components/Drawer/header.js

```diff
@@ -15,7 +15,7 @@ export default function Header({ content, id }) {
 
     return (
         <>
-            {content}
+            <StyledTitle>{content}</StyledTitle>
```

Comment:
why do we need to add a styled here?
This is not necessary because the user can pass a string or any node element that they want on the header,
if you pass a node you stylize it how you want